packages/contracts-bedrock/deploy/025-ProxyAdminTransferOwnership.ts

```diff
@@ -0,0 +1,50 @@
+import assert from 'assert'
+
+import { ethers } from 'ethers'
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+import { awaitCondition } from '@eth-optimism/core-utils'
+import '@eth-optimism/hardhat-deploy-config'
+import 'hardhat-deploy'
+
+import {
+  getDeploymentAddress,
+  doOwnershipTransfer,
+} from '../src/deploy-utils'
+
+const deployFn: DeployFunction = async (hre) => {
+  const proxyAdmin = await getDeploymentAddress(hre, 'ProxyAdmin')
+
+  // If we have the key for the controller then we don't need to wait for external txns.
+  // Set the DISABLE_LIVE_DEPLOYER=true in the env to ensure the script will pause to simulate scenarios
+  // where the controller is not the deployer.
+  const isLiveDeployer = await liveDeployer({
```

Comment:
I don't think we need the concept of a live deployer anymore because these will always be new systems, never upgrades that want to keep the owners as multisigs